config.md

```diff
@@ -57,11 +57,11 @@ For Windows, see links for details about [mountvol](http://ss64.com/nt/mountvol.
   This value MUST be an absolute path.
   * Windows: one mount destination MUST NOT be nested within another mount (e.g., c:\\foo and c:\\foo\\bar).
   * Solaris: corresponds to "dir" of the fs resource in zonecfg(8).
-* **`type`** (string, REQUIRED) The filesystem type of the filesystem to be mounted.
+* **`type`** (string, OPTIONAL) The filesystem type of the filesystem to be mounted.
```

Comment:
This probably needs an `omitempty` [here][1] and dropping the `required` entry [here][2].

[1]: https://github.com/opencontainers/runtime-spec/blob/2d491b03fa3c09680c4b557b74825e0db6b04a7b/specs-go/config.go#L116
[2]: https://github.com/opencontainers/runtime-spec/blob/2d491b03fa3c09680c4b557b74825e0db6b04a7b/schema/defs.json#L145